tests/helper/test_helper.py

```diff
@@ -152,24 +152,34 @@ def test_index_starred(
 
 
 @pytest.mark.parametrize(
-    "mentioned_messages",
+    "mentioned_messages, wildcard_mentioned_messages",
     [
-        {537286, 537287, 537288},
-        {537286},
-        {537287},
-        {537288},
-        {537286, 537287},
-        {537286, 537288},
-        {537287, 537288},
+        ({537286, 537288}, {537287}),
+        ({537286}, {537286}),
+        ({537287}, {537286, 537288}),
+        ({537288}, {537286, 537287, 537288}),
+        ({537286, 537287}, set()),
+        ({537286, 537288}, {537286, 537288}),
+        ({537287, 537288}, {537286, 537287, 537288}),
```

Comment:
We've lost some test examples here of just mentions; that may not be critical, but even if we just add these mixed cases then adding inline test ids would be helpful to describe what situation each is covering without looking at the values. You may see when reading those ids that you're missing some combinations that might be interesting, such as only-wildcard.